packages/cli/Dockerfile

```diff
@@ -18,13 +18,14 @@ RUN apt-get install -y nodejs
 RUN npm install sharp@0.33.0
 
 # Install the landing assets
+COPY ./basemaps-cli-config*.tgz /app/
```

Comment:
I don't think we need to install this in the docker. Please test after remove it.